Source/KSCrash/Recording/Tools/KSLogger.h

```diff
@@ -259,6 +260,24 @@ bool kslog_setLogFilename(const char* filename, bool overwrite);
 /** Clear the log file. */
 bool kslog_clearLogFile();
 
+typedef void (*KSLogFunction)(const char* level, const char* file,
+                     int line, const char* function,
+                     const char* fmt, va_list args);
+
+/** Set the function responsible for logging.
+ *
+ * When set, will deliver all logging to this function
+ * instead of the default handler, so stdout output and file
+ * output will be disabled unless the handler has its own
+ * mechanism for dealing with those.
+ *
+ * WARNING: Only call async-safe functions from this function!
```

Comment:
Added a warning here, like for other callbacks that happen after a potential crash